__test__/entrypoint/controllers/sample/createSample.controller.test.ts

```diff
@@ -0,0 +1,28 @@
+import { Application } from 'express';
+import supertest from 'supertest';
+import { Env } from '../../../../src/infrastructure/env';
+import { Server } from '../../../../src/server';
+
+// jest.mock('../../../../src/infrastructure/middlewares/registryMessage');
```

Comment:
I think those mocks are not in use. Maybe we could remove them.

Reply:
Comment removed.